src/components/ValidatedInputField/index.js

```diff
@@ -32,6 +34,8 @@ ValidatedInputField.propTypes = {
   type: PropTypes.string,
   label: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Why the eslint-disable?

Reply:
I didn't know how to add default-prop for a function, but it seems to work by just adding an empty string. If that's all right to do :)

Reply:
You could add empty string as the default, but since the actual prop is expected to be a function, better not to set default prop to some other type (string). I think type checking is also done after resolving default props, so you will get some warnings if type is changing. Usually I just use "null" as the default for functions that are not required. Other options is to use function that does nothing: `onChange: () => {}`

Reply:
Okay, thanks. Changed to null.